extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java

```diff
@@ -105,10 +113,11 @@ IndexDependencyBuildItem indexLiquibase() {
     @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
     @Record(STATIC_INIT)
     void nativeImageConfiguration(
-            LiquibaseRecorder recorder,
+            LiquibaseRecorder ignored,
```

Comment:
I think you can drop this parameter and the `@Record` annotation if we don't need them.